packages/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -391,6 +396,31 @@ abstract contract StandardBridge is Initializable {
         });
     }
 
+    /// @notice Initiates a withdrawal from L2 to L1 to a target account on L1.
+    /// @param _localToken Address of the token on this chain.
+    /// @param _remoteToken Address of the corresponding token on the remote chain.
+    /// @param _from       Address of the sender.
+    /// @param _target     Address of the receiver.
+    /// @param _amount     Amount of the token sent.
+    function _initiateBridgeETHERC20(
```

Comment:
I am not a solidity developer, but it feels a bit wrong to me that this function _should_ only ever be invoked with the L1 Eth ERC20 token address as local token, and the zero address as remote token.

Maybe it's more idiomatic to leave it as written, but if we know the required call parameters, why even include them as parameters to this function?

Minimally, could we note in the doc what the required values for these parameters are, or perhaps check them explicitly?

Reply:
We can hardcode the `_remoteToken`. For the `_localToken` address, it can be any ERC20 token address on L2.

Reply:
Maybe this is my misunderstanding.  When is this function intended to be called?  I thought it was only invoked when the L2 gas token is not the L1 native gas token, and the token being transferred was the wrapped version of the L1 native ETH?  Is that not the case?

Reply:
Yes, the wrapped version of the L1 native ETH can be any addresses on L2, so we don't know the `_localToken` address. We can read this address from the `L1Block` contract.